packages/xsnap/src/xsnap.js

```diff
@@ -152,7 +152,7 @@ export async function xsnap(options) {
       sourceStream.pipe(destStream, { end: false });
 
       done = finished(sourceStream);
-      done.catch(noop).then(() => sourceStream.unpipe(destStream));
+      done.catch(noop).then(() => sourceStream.unpipe(destStream), noop);
```

Comment:
Huh, I'm guessing we needed the errback (second arg) in the `.then()` because the linter doesn't believe that `noop` will never throw? But, why didn't it complain that the promise returned by `.then` is similarly ungrounded?

@mhofman is the expert on this code, but I'm guessing that a `void` in front of the whole thing would be reasonable, and if that addresses whatever linter complaint that prompted the extra `noop` inside the `then()`, it might be a better fix overall.

Reply:
That extra `noop` doesn't change a thing since we already have a `.catch(noop)` before that. And yeah it's more likely the linter would complain about `sourceStream.unpipe(destStream)` throwing (which it doesn't in practice). We can void the resulting promise if needed in this case.